src/db.rs

```diff
@@ -0,0 +1,52 @@
+use tokio_postgres::{NoTls, Error};
+use dotenv::dotenv;
+use std::env;
+
+pub async fn insert_user(name: &str, age: i32) -> Result<(), Error> {
+    dotenv().ok();
```

Comment:
Haven't explored this yet but there's probably a better place for this? I can't imagine doing this in every single function.